stdlib/websub/src/main/ballerina/src/websub/subscriber_service_endpoint.bal

```diff
@@ -35,24 +35,55 @@ public type Listener object {
 
     private http:Listener? serviceEndpoint = ();
 
+    # Creates a new `websub:Listener`.
+    #
+    # + port - The port number of the remote service
+    # + config - The configurations related to the `websub:Listener`
     public function __init(int port, SubscriberListenerConfiguration? config = ()) {
         self.init(port, config);
     }
 
+# Binds a service to the `websub:Listener`.
+# ```ballerina
+# error? result = websubListener.__attach(helloService);
+# ```
+#
+# + s - Type descriptor of the service
+# + name - Name of the service
+# + return - `()` or else an `error` upon failure to register the listener
     public function __attach(service s, string? name = ()) returns error? {
         // TODO: handle data and return error on error
         externRegisterWebSubSubscriberService(self, s);
     }
 
+# Stops consuming messages and detaches the service from the `websub:Listener`.
+# ```ballerina
+# error? result = websubListener.__detach(helloService);
+# ```
+#
+# + s - Type descriptor of the service
+# + return - `()` or else an `error` upon failure to detach the service
     public function __detach(service s) returns error? {
     }
 
+# Starts the `websub:Listener`.
+# ```ballerina
+# error? result = websubListener.__start();
+# ```
+#
+# + return - `()` or else an `error` upon failure to start the listener
```

Comment:
```suggestion
# + return - `()` or else an `error` upon failure to start the listener
```
Need to specify the error.

Reply:
We are using a common error in this module. So we don't need to specify the error.